Components/Transform/ViewOps/ViewOps/ALegendre/Builder.hpp

```diff
@@ -199,10 +203,18 @@ void builder(Tview opView, const Evector<Tdata>& grid,
       scaling = Evector<ScalarType>::LinSpaced(L, 0, L - 1);
       scaling = scaling.array() * (scaling.array() + 1.0);
    }
-   else if constexpr (LlDiff != 0)
+   else if constexpr (LlDiff > 0)
+   {
+      auto L = opView.dims()[LIdx];
+      scaling = Evector<ScalarType>::LinSpaced(L, 0, L - 1);
+      scaling = (scaling.array() * (scaling.array() + 1.0)).pow(LlDiff);
+   }
+   else if constexpr (LlDiff < 0)
    {
       auto L = opView.dims()[LIdx];
       scaling = Evector<ScalarType>::LinSpaced(L, 0, L - 1);
+      // avoid division by zero
```

Comment:
This is computing 1/l(l+1) but for l = 0 it should set the value to zero right?

Reply:
yes, but IIRC somewhere is used to divide, hence the epsilon. I remember doing this after finding a `nan` in a operator which was passing the tests, but I don't remember the details.